src/components/universal/core/Stat/Stat.recipe.ts

```diff
@@ -0,0 +1,97 @@
+import { defineSlotRecipe } from "@pandacss/dev";
+
+export const statRecipe = defineSlotRecipe({
+  className: "stat",
+  description: "The styles for the Stat component",
+  slots: ["root", "label", "value", "helpText"],
+  base: {
+    root: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "start",
+      borderRadius: "md",
+      boxShadow: "sm",
+      bgColor: "accent.default",
```

Comment:
Would change this to `bg.default`.... also would add the following styles:

```ts
 borderWidth: "1px",
 borderColor: "border.default",
 ```
 
 I think we will want to opt into the accent `bgColor` if we need more focus on the stats.

Reply:
I'm good with this. The accent.default looked a bit tacky anyways, but we were using it for every other component default.

Reply:
Yeah I think it is great for `Button` or `Accordion` or anything that has like a "call to action" where it shows that component is pressable (at least for our branding). For data display, not necessarily the best default option, imo!